### DISCUSSION TITLE
How to Build

### DISCUSSION THREAD

**Discussion by rchive1** [chuongmep/CadPythonShell#13] [Q&A]
@chuongmep, can you give a brief description of how to build this? It looks like you used Nuke in Visual Studio, is that correct? It appears Nuke is undergoing some maintenance currently, so I can't download it for VS or VS Code. I was wondering if I can build manually using `dotnet` in the Windows command prompt? If it's too complicated, I'll wait for Nuke to be available again.

Thanks!

**Comment by chuongmep** [comment]
Hi @rchive1,

The project build process is separate from the deployment process. Nuke is a tool used to package the project into an MSI file. When cloning the project, ensure you have Nuke installed on your machine to package the project. You can install [Nuke ](https://nuke.build/docs/getting-started/installation/)globally using the following command:

```bash
dotnet tool install Nuke.GlobalTool --global
```
After installation, you can build the project using the command:
```
nuke
```
Let me know if you have any questions or if I can assist you further.
